rubicon_ml/intake_rubicon/viz.py

```diff
@@ -18,7 +21,6 @@ def __init__(self, metadata=None, **catalog_data):
 
     def _get_schema(self):
         """Creates an Experiments Table visualization and sets it as the visualization object attribute"""
-        from rubicon_ml.viz import ExperimentsTable
```

Comment:
can you move this import back into the `ExperimentsTableDataSource._get_schema` function?

can you also move the `from rubicon_ml.viz import DataframePlot` into `DataframePlotDataSource._get_schema` and `from rubicon_ml.viz import MetricCorrelationPlot` into `MetricCorrelationPlotDataSource._get_schema` (missed that when I reviewed @austinhk 's PR)

if we import them at the top level, it makes the viz dependencies required for the core package import, which we don't want. when we move them inside the relevant `_get_schema`, they'll only be imported when the visualizations are loaded from a catalog